source/03-components/call-to-action/_call-to-action.scss

```diff
@@ -0,0 +1,24 @@
+// @file
+// Component: c-call-to-action
+
+@use '00-config' as *;
+
+.c-call-to-action {
+  .l-grid {
+    align-items: center;
+  }
+}
+
+.c-call-to-action__media {
+  @include breakpoint(gesso-breakpoint('tablet')) {
+    grid-column-start: 2;
+    order: 2;
+  }
+}
+
+.c-call-to-action__copy {
```

Comment:
@jbwagner Maybe call this 'text' instead of 'copy', just for consistency with other components?

Reply:
Changed. Naming things is hard. [437f903](https://github.com/forumone/gesso/pull/886/commits/437f9034add8a9ca842686fb7f10b48fe5548223)